### PR TITLE
[master] Remove `types.bal` generation from low level service

### PR DESCRIPTION
**Pull request by lnash94**
Related issue: https://github.com/wso2-enterprise/internal-support-ballerina/issues/528

## Automation tests
 - Unit tests - added
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java [hunk 1]
```diff
@@ -1102,4 +1106,37 @@ public static boolean isIntegerSchema(Schema<?> fieldSchema) {
...
+        }
+        return type;
+    }
+
+    private static String getPathParameterType(Schema<?> typeSchema, String pathParam)
+            throws BallerinaOpenApiException {
+        String type;
+        if (!(isStringSchema(typeSchema) || isNumberSchema(typeSchema) || isBooleanSchema(typeSchema)
+                || isIntegerSchema(typeSchema))) {
+            type = "string";
```
Comment (NipunaRanasinghe) [inline openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java:1134]: Don't we have a common constant for Ballerina string type?

File: openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenApiGenServiceCmdTest.java [hunk 2]
```diff
@@ -111,4 +113,34 @@ public void testOneOfSchemaGen() throws IOException {
...
+        String expectedService = expectedServiceLines.collect(Collectors.joining("\n"));
+        expectedServiceLines.close();
+        Assert.assertFalse(Files.exists(this.tmpDir.resolve("types.bal")));
+        if (Files.exists(this.tmpDir.resolve("withoutdatabinding_service.bal"))) {
+            String generatedService = getStringFromFile(this.tmpDir.resolve("withoutdatabinding_service.bal"));
+            Assert.assertEquals(replaceWhiteSpace(generatedService), replaceWhiteSpace(expectedService),
+                    "Expected content and actual generated content is mismatched for: " + yamlPath);
+            deleteGeneratedFiles("without-data-binding-service.bal");
+        } else {
+            Assert.fail("Service generation for All Of Schema type failed.");
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenApiGenServiceCmdTest.java:143]: is this error message correct?
```suggestion
            Assert.fail("Service generation for All Of Schema type failed.");
```

Comment (lnash94) [reply openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenApiGenServiceCmdTest.java]: This is a mistake and updated according to the test.

File: openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java (hunk 1 above)
Comment (lnash94) [reply openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java]: Updated with the constant value [cf982ff](https://github.com/ballerina-platform/openapi-tools/pull/1587/commits/cf982ffe6f3bfdbdce1dcd6b02ba6aae0acfc952)

**Review by TharmiganK — APPROVED** [review]

**Pull request merged**